src/main/groovy/wooga/gradle/snyk/cli/SnykTaskSpec.groovy

```diff
@@ -87,4 +96,20 @@ trait SnykTaskSpec extends CommonArgumentSpec {
         snykPath.set(value)
     }
 
+    private final Property<String> workingDirectory = objects.property(String)
+
+    /**
+     * To override the working directory when invoking the tool
+     * @return
+     */
+    @Input
+    @Optional
+    Property<String> getWorkingDirectory() {
+        workingDirectory
+    }
+
+    void setWorkingDirectory(Provider<String> value) {
+        workingDirectory.set(value)
+    }
+
```

Comment:
Is this really still needed?
It should be a Directory instead of String and only marked as Internal. But I honestly don't see the need for it anymore.

Reply:
I agree it's no longer needed. I will removei t.